parts/masterparams.t

```diff
@@ -29,6 +29,13 @@
   {{end}}
 {{end}}
 {{if IsHostedMaster}}
+    "masterSubnet": {
```

Comment:
Could we change this param to `hostedMasterSubnet` from `masterSubnet`, because it appears to be intended only in "hosted master" scenarios?

Reply:
I would vote for not making those kinds of specializations. masterSubnet is already being used thoughout (and in line 27) above as well. I don't see a use case regarding why this param should be called by a different name.

Reply:
I didn't catch that, thanks! It would help if the `if not IsHostedMaster` was coupled with an `else` clause (or `if IsHostedMaster`--`else`) to more clearly communicate that only one of those branches are entered into during a single execution.

Reply:
That remark was intended for another PR, not this one!